docs/current_docs/api/constructors.mdx

```diff
@@ -37,19 +37,6 @@ Here is an example module with a custom constructor:
 :::info
 In the Python SDK, the [`@dagger.object_type`](https://dagger-io.readthedocs.io/en/latest/module.html#dagger.object_type) decorator wraps [`@dataclasses.dataclass`](https://docs.python.org/3/library/dataclasses.html), which means that an `__init__()` method is automatically generated, with parameters that match the declared class attributes.
 :::
-
-The code listing above is an example of an object that has typed attributes.
-
-If a constructor argument needs an asynchronous call to set the default value, it's
-possible to replace the default constructor function from `__init__()` to
-a factory class method named `create`, as in the following code listing:
-
-:::warning
-This factory class method must be named `create`.
-:::
-
-```python file=./snippets/constructors/python/async/main.py
-```
```

Comment:
Why removing that?

Reply:
It is not standard but it may be needed for more complex use cases or if you need any async call in the constructor.

So it think it should be somewhere, but not under "Simple Constructor". My concern is people using this even for simpler use cases or cases that are covered by the standard dataclass.

Reply:
Added this back